matlab-src/whole_body_model_functions/wbm_getWorldFrameFromFixedLink.m

```diff
@@ -0,0 +1,33 @@
+function [ world_R_base,world_p_base ] = wbm_getWorldFrameFromFixedLink( varargin )
+%WBM_GETWORLDFRAMEFROMFIXEDLINK returns the floating base position for a
+%given reference link name and joint configuration
+%
+%   Arguments :
+%       Optimised Mode : link_name - string matching URDF name of the link (frame)
+%       Normal Mode : link_name - string matching URDF name of the link (frame)
+%                      qj - joint configuration ( (NumDoF x 1))
+%   Returns : R - rotation from rootLink to world frame (3 x 3)
+%             p - translation from rootLink to world frame (3 x 1) 
+%
+% Author : Naveen Kuppuswamy (naveen.kuppuswamy@iit.it)
+% Genova, Dec 2014
+
+    switch(nargin)
+        case 1 
+            qT = wbm_forwardKinematics(varargin{1});
```

Comment:
This is computing the world_H_referenceLink = world_H_base * base_H_referenceLink transform , where world_H_base is the state already setted in the wbm , right ? So if the stored world_H_base is not identity this is not going to work?